.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+
+main: &main
+    machine: true
+    environment:
+      # Used by qthelpers to close widgets after a defined time
+      - TEST_CI: "True"
+      - TEST_CI_APP: "True"
+    steps:
+      - checkout
+      - run:
+          command: docker pull dorowu/ubuntu-desktop-lxde-vnc:trusty
+      - run:
+          name: Install required pachages
+          command: ./.circleci/install.sh
+      - run:
+          name: Run tests
+          command: ./.circleci/run-tests.sh
+      - store_test_results:
+          path: test-reports
+      - store_artifacts:
+          path: test-reports
```

Comment:
I see test-reports is used here. Is this really necessary?

Reply:
I think this uploads the test results to CircleCI but it is not clear what they do with it, so I'll remove this.